pycodestyle.py

```diff
@@ -1554,6 +1561,8 @@ def check_logical(self):
         if self.logical_line:
             self.previous_indent_level = self.indent_level
             self.previous_logical = self.logical_line
+            if not self.indent_level:
+                self.previous_logical_toplevel = self.logical_line
```

Comment:
This name is a bit poor. We're also now adding a new attribute to a Checker that a plugin can request. Given that we're exposing this to plugins, can we consider a better name?